### PR TITLE
Docs 5212 add info about adding contracts pa

### PR DESCRIPTION
**Pull request by priyaayyar**
Added a link to information about how to request access in Exchange. Also, clarified that requests are made from Exchange and not API Manager. The Exchange documentation will also be updated by Will Warner accordingly.

### CODE REVIEW

File: modules/ROOT/pages/troubleshooting-archetype-error-when-creating-policy.adoc [hunk 1]
```diff
@@ -1,10 +1,11 @@
-= Archectype error when creating a policy
+= Archectype error when creating policies
```
Comment (nahuel-dallavecchia) [inline modules/ROOT/pages/troubleshooting-archetype-error-when-creating-policy.adoc]: Typo in arche*c*type -> archetype.

Comment (nahuel-dallavecchia) [reply modules/ROOT/pages/troubleshooting-archetype-error-when-creating-policy.adoc]: Why the plural? An archetype can create one policy at a time.

File: modules/ROOT/pages/manage-client-apps-latest-task.adoc [hunk 2]
```diff
@@ -27,3 +27,6 @@ To manage your API Instance contracts:
 This tab displays information about the app, access controls for removing the app (revoking access), resetting the client ID and client secret, and adding or removing app owners:
```
Comment (nahuel-dallavecchia) [inline modules/ROOT/pages/manage-client-apps-latest-task.adoc]: Client ID cannot be reset.

**Review by nahuel-dallavecchia — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/manage-client-apps-latest-task.adoc (hunk 2 above)
Comment (priyaayyar) [reply modules/ROOT/pages/manage-client-apps-latest-task.adoc]: Fixed! Thanks!

File: modules/ROOT/pages/manage-client-apps-latest-task.adoc [hunk 3]
```diff
@@ -3,7 +3,7 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-In API Manager, you can grant access to applications that want to consume your APIs by creating instance Contracts. +
+After you grant access to applications in Exchange, a contract is automatically created for the API instance. You can then manage these contracts from API Manager. +
```
Comment (nahuel-dallavecchia) [inline modules/ROOT/pages/manage-client-apps-latest-task.adoc]: "After you grant access to applications in Exchange" is a bit confusing. The API Owner may grant access to a client application request in the API in *API Manager*. The client application owner must request access to the API in the Exchange asset corresponding to the API, in *Exchange*.

Comment (nahuel-dallavecchia) [inline modules/ROOT/pages/manage-client-apps-latest-task.adoc]: "You can then manage these contracts from API Manager" May be from your API in API Manager? Contracts "live" within APIs.

**Review by nahuel-dallavecchia — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/manage-client-apps-latest-task.adoc (hunk 3 above)
Comment (priyaayyar) [reply modules/ROOT/pages/manage-client-apps-latest-task.adoc]: Fixed! Thanks!

Comment (priyaayyar) [reply modules/ROOT/pages/manage-client-apps-latest-task.adoc]: Thanks, Nahuel! I think that's too much detail that is not required here. I simplified it a bit more.

**Review by hannanelson — APPROVED** [review]
LGTM

**Review by nahuel-dallavecchia — APPROVED** [review]

**Pull request merged**